docs/advanced.rst

```diff
@@ -53,3 +53,8 @@ access control. For example:
     def check_access():
         if request.args.get("channel") == "analytics" and not g.user.is_admin():
             abort(403)
+
+    app.register_blueprint(sse, url_prefix='/sse')
+
+Please note: When defining a :meth:`~flask.Blueprint.before_request` handler,
```

Comment:
You could use the `.. warning::` directive to make this look more visually distinctive! See https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-warning

Reply:
Thanks! I've updated the pull request to take your suggestions into account. :)